spec/index.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Spec proposal</title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"></script>
+    <script class='remove'>
+      // See https://github.com/w3c/respec/wiki/ for how to configure ReSpec
+      var respecConfig = {
+        specStatus: "CG-DRAFT", // FIXME: not a CG-DRAFT, ED?
+        shortName: "lws-ucs",
+        copyrightStart:  "2024",
+        // previousPublishDate:  "N/A",
+        editors: [
+          { name: "Pierre-Antoine Champin" },
+          { name: "Hadrian Zbarcea" },
+        ],
+        group: "solid",  // FIXME: https://respec.org/w3c/groups/ does not list 'lws' yet
```

Comment:
I was surprised to discover that this list of groups [requires a manual update](https://github.com/speced/respec-web-services/issues/395)... I requested it.

Fortunately, we don't need to wait for that. "Qualified" group names seem to work.
```suggestion
        group: "wg/lws",
```